<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1405</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_desenhar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Desenhar Polígono</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reset">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_novoPoligono">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Novo Polígono</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_opcoes">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_sair">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>390</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
   <widget class="Tela" name="Frame">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>681</width>
      <height>591</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="Translacao">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>151</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Objeto</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="text">
      <string>Translação </string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::PlainText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>dX</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="caixa_trans_dx">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>60</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-1000</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>dY</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="caixa_trans_dy">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>100</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-1000</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_aplicar_translacao">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Translação</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Escala">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>151</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Objeto</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Escala</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Fator X</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="caixaDupla_escala_sx">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>62</width>
       <height>25</height>
      </rect>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Fator Y</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="caixaDupla_escala_sy">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>62</width>
       <height>25</height>
      </rect>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_aplicar_escala">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Escala</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Rotacao">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>380</y>
      <width>151</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Objeto</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Rotacao</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>angulo</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="caixaDupla_rot_angulo">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>62</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Pivo_X_Y</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="caixa_rot_pivo_x">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <number>5000</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="caixa_rot_pivo_y">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="maximum">
      <number>5000</number>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_aplicar_rotacao">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Rotacao</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget_coordenadas">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>500</y>
      <width>231</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="ControledeCamera">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>30</y>
      <width>181</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Window</string>
    </property>
    <widget class="QDoubleSpinBox" name="spin_wc_x">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>10000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>10.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spin_wc_y">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-100000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>10.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spin_zoom">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <double>10.000000000000000</double>
     </property>
     <property name="maximum">
      <double>20000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>50.000000000000000</double>
     </property>
     <property name="value">
      <double>10.000000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spin_angulo">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
     <property name="value">
      <double>5.000000000000000</double>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_atualizar_visualizacao">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Atualizar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Distancia</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Angulo</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1405</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Tela</class>
   <extends>QFrame</extends>
   <header location="global">tela.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btn_sair</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
